File: Comparacao_Proc.java
public class Soma {
    public static void main(String[] args) {
        long sum = 0;
        long N = 1000000000L; // 1 bilhão

        long start = System.currentTimeMillis();

        for (long i = 1; i <= N; i++) {
            sum += i;
        }

        long end = System.currentTimeMillis();

        System.out.println("Soma = " + sum);
        System.out.printf("Tempo gasto: %.3f segundos\n", (end - start) / 1000.0);
    }
}

File: Comparacao_Proc.py
import time

sum = 0
#N = 1000000000  # 1 bilhão
N = 1000000000

start = time.time()

for i in range(1, N+1):
    sum += i

end = time.time()

print(f"Soma = {sum}")
print(f"Tempo gasto: {end - start:.3f} segundos")